weather.py

```diff
@@ -35,7 +41,10 @@ def convert_f_to_c(temp_in_farenheit):
     Returns:
         A float representing a temperature in degrees celcius, rounded to 1dp.
     """
-    pass
+    temp_in_farenheit = float(temp_in_farenheit)
+    temp_in_celcius = float((temp_in_farenheit-32)*5/9)
+    result = round(temp_in_celcius, 1)
```

Comment:
Since `temp_in_farenheit` is already a `float`, `temp_in_celcius` should automatically be a `float` as well, so you won't have to cast `float()` again.